Makefile.devel

```diff
@@ -334,26 +318,25 @@ tag:
 		rm $$f.old; \
 	   done) \
 	fi
```

Comment:
This if-then-else (starting with `if [ $(PRERELEASE_TAG) = $(VERSION) ]; then`) should be commented-out as the PG website is now handled separately.